.github/workflows/security.yml

```
@@ -0,0 +1,107 @@
+      # Snyk security scan
+      - name: Run Snyk to check for vulnerabilities
+        uses: snyk/actions/node@master
+        continue-on-error: true
+        env:
+          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+        with:
+          args: --severity-threshold=high
+          
+      # Dependency Review to identify vulnerable dependencies
+      - name: Dependency Review
+        uses: actions/dependency-review-action@v3
```

actionlint (1.7.7)

the runner of "actions/dependency-review-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)